Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -45,6 +45,8 @@ def http_request(method, url_suffix, params={}, data=None):
                 f'Error in API call to Sentinel One [{res.status_code}] - [{res.reason}] \n'
                 f'Error details: [{errors}]'
             )
+        except ValueError as value_error:
+            raise value_error
```

Comment:
I think that they meant that no matter what the error is, don't just return the error but add details about it.
Do you understand what I mean?